gatsby-node.js

```diff
@@ -120,6 +124,16 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
           }
         }
       }
+      singleWorkshop: allMdx(

```

Comment:
Filter out the `workshop` collection like others

Reply:
Its throwing the same error

```
warn The GraphQL query in the non-page component "/mnt/d/GitHub/layer5/src/templates/workshop-single.js" will not be run.
Exported queries are only executed for Page components. It's possible you're
trying to create pages in your gatsby-node.js and that's failing for some
reason.

If the failing component(s) is a regular component and not intended to be a page
component, you generally want to use a <StaticQuery> (https://gatsbyjs.org/docs/static-query)
instead of exporting a page query.

If you're more experienced with GraphQL, you can also export GraphQL
fragments from components and compose the fragments in the Page component
query and pass data down into the child component — https://graphql.org/learn/queries/#fragments
```

Reply:
I can't able to filter out the `workshop` collection like the others. For some reason its throwing the above error again and again.

Reply:
Can you write here what you are writing?

Reply:
```js
  const singleWorkshop = allNodes.filter(
      node => node.fields.collection === `workshops`
  );
```

Reply:
Gents, I'm moving forward with a merge here, so that we can iterate quickly toward go-live. Please do file an issue if a follow up is needed.  The "Reference in new issue" action is handy for this.

I don't see these errors when I checkout the latest version and build.